bundles/AdminBundle/Controller/Admin/Document/PageController.php

```diff
@@ -244,26 +244,32 @@ public function displayPreviewImageAction(Request $request)
     public function checkPrettyUrlAction(Request $request)
     {
         $docId = $request->get('id');
-        $path = trim($request->get('path'));
+        $path = (string) trim($request->get('path'));
         $path = rtrim($path, '/');
```

Comment:
can you move rtrim after empty path check? otherwise path "/" will be valid now.

Reply:
Current validation after changes:

Input: `/`,  `//` or more repeating... This is not perfect, might want to add additional error message when the string only contains unwanted characters. But it flags an error at the very least.
Error: `URL must be at least 2 characters long.`

Input: `a`
Error: `URL must start with /.`
`URL must be at least 2 characters long.`

Input: `ab`
Error: `URL must start with /.`

Input: `/ab`
*Success*

Reply:
It´s just a warning for the user....looks good to me!